test/utils.py

```diff
@@ -1394,3 +1394,19 @@ def get_response_data(response):
     assert 'errors' not in response_json, response_json
 
     return response_json['data']
+
+
+def get_tarantool_installer_cmd(package_manager):
+    short_version = tarantool_short_version()
+    major, minor = short_version.split(".")
+
+    if int(major) >= 2 and int(minor) > 8:
+        installer = "https://tarantool.io/release/2/installer.sh"
+        is_prerelease = any(postfix in tarantool_version() for postfix in ["-alpha", "-beta", "-rcc"])
+        if is_prerelease:
+            installer = "https://tarantool.io/pre-release/2/installer.sh"
```

Comment:
Comments above (3.0.0 and single URL) are applicable here too.

Reply:
Fixed